requirements.txt

```
@@ -1,4 +1,4 @@
-retworkx>=0.10.1
+retworkx @ git+https://github.com/georgios-ts/retworkx@pr-dijkstra-visitor
```

Now that retworkx 0.11.0 is released we can remove the on hold and change this to:

```suggestion
retworkx>=0.11.0
```

Also I think we should add a release note about bumping the minimum retworkx version to 0.11.0

---

(since this comment, #7665 merged which bumped the `retworkx` dependency to 0.11 already)